src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2437,66 +2435,65 @@ private void purgeCustomExportFormats(int from) {
 
     @Override
     public PreviewPreferences getPreviewPreferences() {
-        if (this.previewPreferences == null) {
-            updatePreviewPreferences();
+        if (Objects.nonNull(previewPreferences)) {
```

Comment:
Usage of Objects.nonNull is only useful in predicates/lambdas:
>    This method exists to be used as a Predicate, filter(Objects::nonNull)

Reply:
We already used this method in multiple locations in JabRef and everywhere in the preference conversion complex. For consistency in the code I'd like to continue to use it. The api-note, that this method especially exists for use as a Predicate doesn't mean that it couldn't be used for something else too. 😅